src/Microsoft.DotNet.Core.Build.Tasks/GenerateDepsFile.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Microsoft.Extensions.DependencyModel;
+using Newtonsoft.Json;
+using NuGet.Frameworks;
+using NuGet.ProjectModel;
+
+namespace Microsoft.DotNet.Core.Build.Tasks
+{
+    /// <summary>
+    /// Generates the $(project).deps.json file.
```

Comment:
`Generates the deps.json file at the path specified by DepsFilePath` ?

I agree 99% of the time it will be true, but only because of assumptions on how the levers this unit of code exposes will be used. 

Like I said this is a nit, I just think it will make the expectations of this unit of code a little clearer.

---
In reply to: [73044051](https://github.com/dotnet/cli/pull/3962#discussion_r73044051) [](ancestors = 73044051)